src/Common/src/System/Net/Logging/HttpEventSource.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+using System.Net.Http;
+using System.Globalization;
+
+namespace System.Net
+{
+    [EventSource(Name = "Microsoft-System-Net-Http",
+        Guid = "bdd9a83e-1929-5482-0d73-2fe5e1c0e16d",
+        LocalizationResources = "FxResources.System.Net.Http.SR")]
+    internal sealed class HttpEventSource : EventSource
+    {
+        private const int ASSOCIATE_ID = 1;
```

Comment:
Nit: Naming convention: AssociateId, etc (you can run the formatting tool to correct these).